hypothesis/test-pyramid-app
# Add a test Celery task (and admin pages, devdata, and Jinja2)

Add a test Celery task that can be kicked off by an admin page.

This is so that I can kick off this task on production and watch in the Papertrail logs what happens to it when I do a deploy or terminate an instance etc while the task is running.

In order to facilitate this I had to add Celery, admin pages, devdata and Jinja2 templates to this project.

## Testing notes

 * `make services devdata`
 * `make sure`
 * `make dev`
 * Visit: http://localhost:9800/admin
 * Press "Work"
 * Observe the terminal for lines of logging
